### PR TITLE
Add new york zoos

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: src/chrome/content/rules/Bronxzoo.com.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Bronxzoo.com">
+	<target host="bronxzoo.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Bronxzoo.com.xml]: Please see also:
www.bronxzoo.com
app.bronxzoo.com
apps.bronxzoo.com
pages.bronxzoo.com

File: src/chrome/content/rules/Centralparkzoo.com.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Centralparkzoo.com">
+	<target host="centralparkzoo.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Centralparkzoo.com.xml]: Please see also:
www.centralparkzoo.com
chicks.centralparkzoo.com
pages.centralparkzoo.com

File: src/chrome/content/rules/Prospectparkzoo.com.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Prospectparkzoo.com">
+	<target host="prospectparkzoo.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Prospectparkzoo.com.xml]: Please see also www.prospectparkzoo.com

File: src/chrome/content/rules/Queenszoo.com.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Queenszoo.com">
+	<target host="queenszoo.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Queenszoo.com.xml]: Please see also www.queenszoo.com

**Comment by illicitonion** [comment]
Added where appropriate.

Skipping pages.bronxzoo.com because the http version redirects to bronxzoo.com and the https version has an invalid certificate.

Skipping apps.bronxzoo.com, chicks.centralparkzoo.com, pages.centralparkzoo.com because both the http and https versions are redirects to bronxzoo.com (or for pages.centralparkzoo.com, centralparkzoo.com). If they start being used for something, who knows whether they will be validly https'd, but right now there's no reason for people to be visiting them.

It feels like adding comments saying "Here are some subdomains for which DNS technically exists, but which don't function at all" seems weird and redundant.

**Comment by J0WI** [comment]
@illicitonion we use the documentation for future maintenance on the rules and for the HTTPS Everywhere Atlas.

**Comment by J0WI** [comment]
Thanks and sorry for the delay!

**Pull request merged**